recipes/ixwebsocket/all/conanfile.py

```diff
@@ -56,7 +56,7 @@ def layout(self):
 
     def requirements(self):
         if self.options.get_safe("with_zlib", True):
-            self.requires("zlib/1.2.13")
+            self.requires("zlib/[>=1.2.11 <2]")
         if self.options.tls == "openssl":
             self.requires("openssl/1.1.1s")
```

Comment:
```suggestion
            self.requires("openssl/1.1.1w")
```
or
```suggestion
            self.requires("openssl/[>=1.1 <4]")
```
depending on OpenSSL 3.x support.

Reply:
Good catch, will set it to 1.1.1 as upstream does not seem to actually test v3 by the looks of their issues